File: fjt.pricingservice/MySqlDBModel/rfq_consolidated_mfgpn_lineitem_alternate.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace fjt.pricingservice.MySqlDBModel
{
    [Table("rfq_consolidated_mfgpn_lineitem_alternate")]
    public class rfq_consolidated_mfgpn_lineitem_alternate
    {
        [Key]
        public int id { get; set; }
        [StringLength(50)]
        public string mfgPN { get; set; }
        public int consolidateID { get; set; }
        public int? mfgPNID { get; set; }
        public int? mfgCodeID { get; set; }
        [StringLength(30)]
        public string PIDCode { get; set; }

        public bool isDeleted { get; set; }
        [StringLength(255)]
        public string createdBy { get; set; }
        public DateTime createdAt { get; set; }
        [StringLength(255)]
        public string updatedBy { get; set; }
        public DateTime? updatedAt { get; set; }
        [StringLength(255)]
        public string deletedBy { get; set; }
        public DateTime? deletedAt { get; set; }
        public string customerApproval { get; set; }
        public virtual rfq_consolidated_mfgpn_lineitem rfq_consolidated_mfgpn_lineitem { get; set; }
    }
}

File: fjt.api/models/RFQConsolidatedMFGPNLineItemAlternate.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const RFQConsolidatedMFGPNLineItemAlternate = sequelize.define('RFQConsolidatedMFGPNLineItemAlternate', {

        id: {
            type: Sequelize.INTEGER,
            allowNull: false,
            autoIncrement: true,
            primaryKey: true
        },
        mfgPN: {
            type: Sequelize.STRING,
            allowNull: false
        },
        consolidateID: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        mfgPNID: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        mfgCodeID: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        PIDCode: {
            type: Sequelize.STRING,
            allowNull: true
        },
        isdeleted: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        createdBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        updatedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        deletedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        deletedAt: {
            type: Sequelize.DATE,
            allowNull: true
        },
        customerApproval: {
            type: Sequelize.STRING,
            allowNull: true
        },
        restrictUseInBOMStep: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        restrictUseInBOMWithPermissionStep: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        approvedMountingType: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        }
    }, {
        paranoid: true,
        tableName: 'rfq_consolidated_mfgpn_lineitem_alternate'
    });

    RFQConsolidatedMFGPNLineItemAlternate.associate = (models) => {
        RFQConsolidatedMFGPNLineItemAlternate.belongsTo(models.RFQConsolidatedMFGPNLineItem, {
            as: 'rfqConsolidatedMFGPNLinteItem',
            foreignKey: 'consolidateID'
        });
        RFQConsolidatedMFGPNLineItemAlternate.belongsTo(models.MfgCodeMst, {
            as: 'mfgCodeMst',
            foreignKey: 'mfgCodeID'
        });

        RFQConsolidatedMFGPNLineItemAlternate.belongsTo(models.Component, {
            as: 'component',
            foreignKey: 'mfgPNID'
        });
    };

    return RFQConsolidatedMFGPNLineItemAlternate;
};
